editor-components/content/content13/content13.tsx

```diff
@@ -2,187 +2,101 @@ import * as React from "react";
+    })
+
+    this.addProp({
+      type: "image",
+      key: "image1",
+      displayer: "image1",
+      value: "https://craftohtml.themezaa.com/images/demo-hotel-and-resort-home-01.jpg"
+    })
+
+    this.addProp({
+      type: "image",
+      key: "image2",
+      displayer: "image2",
+      value: "https://craftohtml.themezaa.com/images/demo-hotel-and-resort-home-02.jpg"
+    })
+
+
+
+
   }
 
   getName(): string {
     return "Content 13";
   }
 
   render() {
+
+
+
     return (
       <div className={this.decorateCSS("container")}>
         <div className={this.decorateCSS("max-content")}>
-          <div className={this.decorateCSS("header")}>
-            <span className={this.decorateCSS("subtitle")}>{this.getPropValue("subtitle")}</span>
-            <h2 className={this.decorateCSS("title")}>{this.getPropValue("title")}</h2>
-          </div>
-          <div className={this.decorateCSS("card-child")}>
-            {this.castToObject<Card[]>("content-card").map(
-              
-              (card: Card, index: number) => (
-                <div className={this.decorateCSS("card-item-count")} style={{
-                  width: 90 / this.getPropValue("itemCount") + "%",
-                }}>
-                  <div key={`cnt-4-card-${index}`} className={this.decorateCSS("card")}>
-                    <i>
-                      <img alt="server" src={card.image} />
-                    </i>
-                    <h5 className={this.decorateCSS("card-title")}>{card.title}</h5>
-                    <p className={this.decorateCSS("card-description")}>{card.description}</p>
-                  </div>
+          <div className={this.decorateCSS("title-container")}>
+            <span className={this.decorateCSS("about-title")}>{this.getPropValue("about-title")}</span>
+            <h2 className={this.decorateCSS("maintitle")}>{this.getPropValue("maintitle")}</h2>
+            <p className={this.decorateCSS("description")}>{this.getPropValue("description")}</p>
+            <div className={this.decorateCSS("button-icon")} >
+
+              <button className={this.decorateCSS("about-button")}>{this.getPropValue("buttonText")}</button>
+              <div className={this.decorateCSS("phone-icon")}>
+                <span className={this.decorateCSS("icon")}>📞</span>
+                <h1 className={this.decorateCSS("phone")}>{this.getPropValue("phone")}</h1>
               </div>
-              )
-            )}
+            </div>
+          </div>
+          <div className={this.decorateCSS("right-page")}>
+            <div className={this.decorateCSS("image-container")}>
+              <span className={this.decorateCSS("text")}> Started In</span>
+              <span className={this.decorateCSS("year")}>1995</span>
+              <img src={this.getPropValue("image1")} alt="" className={this.decorateCSS("image1")} />
+              <img src={this.getPropValue("image2")} alt="" className={this.decorateCSS("image2")} />
```

Comment:
image alt should be non-empty.

also this images do not seem right and overflows in desktop and mobile previews.